app/services/search_service.rb

```diff
@@ -67,8 +67,9 @@ def search_tags(query, limit = 10)
     tlist = Tag.where(name: sterms)
       .joins(:node_tag, :node)
       .where('node.status = 1')
```

Comment:
just a small note to sanitize the value in the where statement

Reply:
Thanks for feedback.

First of all this is not part of my change, it was already like this. 
If you think it should be done differently, you can propose the solution.

Second of all passing hash to where method is already safe and user input get sanitized by rails.
You can read up here: https://guides.rubyonrails.org/security.html#injection

Third of all I think node.status values is never set by user and it is not user's input.

Cheers
Mateusz

Reply:
All valid points! Sorry thought that the other queries around it were sanitizing the strings passed to where too so I suggested it for consistency. But looking again that is not the case so taking this back!!